scripts/ci_setup.py

```diff
@@ -378,6 +379,15 @@ def main(args: Any):
     else:
         dotnet.setup_dotnet(args.dotnet_path)
 
+    framework = ChannelMap.get_target_framework_moniker(args.channel)
+    if framework == 'net8.0' or framework == 'nativeaot8.0':
```

Comment:
We don't need to add net6.0, and net7.0 because they are still in their own branches, right?

Reply:
Yes, we will need to keep this in mind when running locally, but we already have to keep the global.json in mind when running locally as most people probably don't run ci_setup.py when running locally.